exampleCourse/questions/examplesRRNGSample/server.py

```diff
@@ -0,0 +1,47 @@
+import random
+import rpy2.robjects as robjects
+
+def generate(data):
+
+  # Pick a seed
+  seed_val = random.randint(1000, 9999)
```

Comment:
Is there any reason to not just use `data["variant_seed"]` here?

Reply:
Lack of knowledge that the `variant_seed` existed. Though, the question is what is the range of the `variant_seed`? Documentation only suggests `Integer`.

https://prairielearn.readthedocs.io/en/latest/devElements/#element-functions

Reply:
This is undocumented, but it's in the range 0 to 2^{32}-1 (inclusive). That is, it's a random unsigned 32-bit integer.

Reply:
Unfortunately, the range of [`Integers`](https://stat.ethz.ch/R-manual/R-patched/library/base/html/integer.html) in _R_ is restricted to `+/-2*10^9`. So, we cannot use the `data["variant_seed"]`. Instead, we have to pre-generate a seed. 

```r
set.seed(2^(32)-1)
# Warning in set.seed(2^(32) - 1) :
#   NAs introduced by coercion to integer range
# Error in set.seed(2^(32) - 1) : supplied seed is not a valid integer
```